integration/meltano-run/validate.sh

```diff
@@ -14,3 +14,7 @@ grep "Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1 cmd_type=command name=dbt-postg
 # we could also run psql statements
 # check for the existance of files
 # etc
```

Comment:
@pandemicsyn you mind giving a quick once-over to this new integration test?

Reply:
👍 lgtm - seems like a good safety check (makes sure `meltano run` is actually tracking state as expected, and that the entry looks roughly legit).